zcash_local_net/src/lib.rs

```diff
@@ -106,82 +115,113 @@ where
     pub fn validator_mut(&mut self) -> &mut V {
         &mut self.validator
     }
-}
 
-impl LocalNet<Zainod, Zcashd> {
-    /// Launch LocalNet.
+    /// Briskly create a local net from configs.
+    /// Here the first parameter is associated with a validator, with the reasoning that an indexer is supported by a validator.
```

Comment:
no need for the waffle here

```suggestion
    /// Launch a local network from validator and indexer configs.
```